www/form/export.js

```diff
@@ -162,13 +150,55 @@ define([
         return csv;
     };
 
-    Export.main = function (content, cb) {
-        var json = Util.clone(content || {});
-        delete json.answers;
-        cb(new Blob([JSON.stringify(json, 0, 2)], {
-            type: 'application/json;charset=utf-8'
-        }));
+    var getFullOrder = function (content) {
+        var order = content.order.slice();
+        var getSections = function (content) {
+            var uids = Object.keys(content.form).filter(function (uid) {
+                return content.form[uid].type === 'section';
+            });
+            return uids;
+        };
+        getSections(content).forEach(function (uid) {
+            var block = content.form[uid];
+            if (!block.opts || !Array.isArray(block.opts.questions)) { return; }
+            var idx = order.indexOf(uid);
+            if (idx === -1) { return; }
+            idx++;
+            block.opts.questions.forEach(function (el, i) {
+                order.splice(idx+i, 0, el);
+            });
+        });
+        return order;
     };
 
+    Export.main = function (content, cb, ext, sframeChan, parsed, zip, sanitize, getUnique, existingNames) {
+        if (sframeChan && content.form) {
+            var _answers = content["answers"];
+            _answers['href'] = parsed.hash;
+            _answers['password'] = parsed.password;
+            _answers['drive'] = true;
+            var answers;
+            sframeChan.query("Q_FORM_FETCH_ANSWERS", _answers, function (err, obj) {
+                answers = obj && obj.results;
+                var rawName = content.metadata.title || 'File';
+                var fileName = getUnique(sanitize(rawName + ' (answers)'), '.json', existingNames);
```

Comment:
Several issues here:
* After a quick test, I ended up with `File (answers)`, which means I didn't know which form it was about
* You're using hardcoded text so only English is supported. You should use translation keys!
* I think it's a bad idea to export 2 files for the same form. The default export is a way to make a backup of your form as a JSON file and be able to restore/re-import it later or in a new form. If we're doing a drive download, I think the "results" file with the questions and all the responses is enough. The drive download is not meant to be re-imported (and cannot be re-imported for most apps)